VMEncryption/main/check_util.py

```diff
@@ -90,7 +94,83 @@ def is_invalid_lvm_os(self):
                     detected = True
         return detected
 
-    def is_precheck_failure(self):
+    def check_kv_url(self, test_url, message):
+        """basic sanity check of the key vault url"""
+        expected = "https://{keyvault-name}.vault.azure.net/"
```

Comment:
update for national clouds  (sorry if duplicate of other review comments)

Reply:
We have removed the strict url checks that would need to be updated for nclouds. We did this because it seemed like it would be difficult to maintain and host agent will be better at catching those errors anyways. So we should instead just work on a mechanism for relaying forward the host agent side errors

Reply:
is the expected string removed or updated?  just caught my eye since the "vault.azure.net" suffix is not expected in national clouds

Reply:
surplus string, now removed, resolving conversation